src/test/java/com/fasterxml/jackson/integtest/gradle/GradleTest.java

```diff
@@ -0,0 +1,186 @@
+import java.util.Arrays;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class GradleTest
+{
+
+    static final String VERSION_QUALIFIER = "SNAPSHOT";
+
+    static final String[] JACKSON_MODULES = {
+            // Core
+            "com.fasterxml.jackson.core:jackson-annotations",
+            "com.fasterxml.jackson.core:jackson-core",
+            "com.fasterxml.jackson.core:jackson-databind",
+
+            // Data Formats
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-avro",
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-cbor",
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-csv",
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-ion",
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-properties",
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-protobuf",
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-smile",
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-xml",
+            "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml",
+
+            // Data Types
+            "com.fasterxml.jackson.datatype:jackson-datatype-guava",
+            "com.fasterxml.jackson.datatype:jackson-datatype-hibernate3",
+            "com.fasterxml.jackson.datatype:jackson-datatype-hibernate4",
+            "com.fasterxml.jackson.datatype:jackson-datatype-hibernate5",
+            "com.fasterxml.jackson.datatype:jackson-datatype-hppc",
+            "com.fasterxml.jackson.datatype:jackson-datatype-jaxrs",
+            "com.fasterxml.jackson.datatype:jackson-datatype-joda",
+            "com.fasterxml.jackson.datatype:jackson-datatype-jdk8",
+            "com.fasterxml.jackson.datatype:jackson-datatype-json-org",
+            "com.fasterxml.jackson.datatype:jackson-datatype-jsr310",
+            "com.fasterxml.jackson.datatype:jackson-datatype-jsr353",
+            "com.fasterxml.jackson.datatype:jackson-datatype-pcollections",
+
+            // JAX-RS
+            "com.fasterxml.jackson.jaxrs:jackson-jaxrs-base",
+            "com.fasterxml.jackson.jaxrs:jackson-jaxrs-cbor-provider",
+            "com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider",
+            "com.fasterxml.jackson.jaxrs:jackson-jaxrs-smile-provider",
+            "com.fasterxml.jackson.jaxrs:jackson-jaxrs-xml-provider",
+            "com.fasterxml.jackson.jaxrs:jackson-jaxrs-yaml-provider",
+
+            // Jackson Jr.
+            // "com.fasterxml.jackson.jr:jackson-jr-all" does not have the bom dependency, see: https://github.com/FasterXML/jackson-databind/issues/2726#issuecomment-707327060
+            "com.fasterxml.jackson.jr:jackson-jr-objects",
+            "com.fasterxml.jackson.jr:jackson-jr-retrofit2",
+            "com.fasterxml.jackson.jr:jackson-jr-stree",
```

Comment:
One more for jackson-jr: `jackson-jr-annotation-support` (added in ... 2.11 I think)

Reply:
The BOM is missing the entry for this: https://github.com/FasterXML/jackson-bom/pull/35

That needs to be merged, then we can add it here.